.env.example

```
@@ -1,95 +1,200 @@
+# Content Caching Configuration
+CONTENT_CACHE_SIZE_GB=8
 
-# CORS & Security
+# GPU Memory Configuration
+GPU_MEMORY_FRACTION=0.95
+
+# =============================================================================
+# CORS & SECURITY
+# =============================================================================
 CORS_ORIGINS=*
 CORS_CREDENTIALS=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CORS_CREDENTIALS key should go before the CORS_ORIGINS key

(UnorderedKey)